interface/src/app/search.service.js

```diff
@@ -0,0 +1,42 @@
+angular.module('adage.search.service', [
+  'ngResource'
+])
+
+.factory('Search', ['$resource', function($resource) {
+  return $resource(
+    '/api/v0/search/',
```

Comment:
Question - should this use the 'ApiBasePath' constant in app/utils.js for standardization?

Reply:
Ooh, nice catch. I'll fix that.

- [ ] use `ApiBasePath` to define REST endpoint